packages/client/src/clients/guide/client.ts

```diff
@@ -310,11 +282,11 @@ export class KnockGuideClient {
 
     switch (event) {
       case "guide.added":
-        return this.addGuide(payload);
+        return this.addOrReplaceGuide(payload);
```

Comment:
Naming nit: would `upsertGuide` make more sense here?

Reply:
Personally "upsert" makes me think in terms of a db operation specifically, so that is probably why - though definitely subjective!